sep-service/ViewModels/Sol.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using sep_service.ViewModels;
+
+namespace sep_service
+{
+    public class Sol
```

Comment:
If possible would love to see here what "Sol" means? I can infer the other view models but this one cannot...

Reply:
It's from the model given to use by the other vendor.  I am not aware of what it means